docs/quickstart.md

```diff
@@ -19,29 +19,18 @@ sidebar_label: Quickstart
 
```

Comment:
We need to update the OpenEBS configuration sequence diagram with MayaStor engine. We can add it to backlog.

Reply:
yes. coming up in new PR.